### DISCUSSION TITLE
我不明白

### DISCUSSION THREAD

**Discussion by match-fire** [share121/inter-knot#1117] [其他]
![](https://i0.hdslb.com/bfs/new_dyn/6b638c3d664b45169f89c46d4616b6de517035667.jpg@1192w.avif)
![](https://i0.hdslb.com/bfs/new_dyn/bce918d312ce2a21bf13ae0e0bd85572517035667.jpg@994w_1930h.avif)
有点不明白

**Comment by inkluv** [comment]
403辣

**Reply by match-fire** [reply]
我重发

**Reply by inkluv** [reply]
我嘞个幼儿园导师

**Comment by su2569** [comment]
是流萤！
